LayoutTests/http/tests/css/link-css-disabled-value-with-slow-loading-sheet-in-error.html

```diff
@@ -20,8 +20,8 @@
 alternateSheetLink.disabled = false;
 
 debug("Testing value of 'disabled' prior to load just after setting them");
-shouldBeNonNull("mainSheetLink.sheet");
-shouldBeNull("alternateSheetLink.sheet");
+shouldBeNull("mainSheetLink.sheet");
+shouldBeNonNull("alternateSheetLink.sheet");
```

Comment:
This test is randomly failing (even before your change).
https://results.webkit.org/?suite=layout-tests&test=http%2Ftests%2Fcss%2Flink-css-disabled-value-with-slow-loading-sheet.html
I think we can remove this line because alternateSheetLink.sheet is checked by the below code. WDTY?

Reply:
Alternatively, maybe we should remove the entire test from the repo sources?

That test is actually out of date with the current spec requirements — and looking through the repo history now, I see it’s a very old test (12+ years), and has been known for more than 9 years now to be flaky.

See https://github.com/WebKit/WebKit/commit/f2f23f38076cd3fad667a68544df4c8b595c9546, which notes both that the test doesn’t even seem to be testing any slowness it was intended to test against (_“These scripts were not "slow" at all”_) and that it also doesn’t align with how WebCore does stylesheet loading (_“WebCore has its own ideas about when to load stylesheets - notably, default stylesheet loading blocks script execution, so we cannot do anything before the stylesheet loads”_).

Further, as far as testing what’s required by the current relevant standards, it’s superseded by these WPT tests:

- https://wpt.fyi/results/css/cssom/HTMLLinkElement-disabled-002.html
- https://wpt.fyi/results/css/cssom/HTMLLinkElement-disabled-001.html

…both of which are tests Safari fails prior to TP181 — but the change in this PR, https://github.com/WebKit/WebKit/pull/19331, makes WebKit now pass.

Finally, per https://github.com/WebKit/WebKit/commit/c6a6d036f285fbaff5be9c601307ce9c5e71ca46 (https://bugs.webkit.org/show_bug.cgi?id=263679) the test was just today marked as `Failure` in the text expectations for yet another platform ([`mac-wk1/TestExpectations`](https://github.com/WebKit/WebKit/commit/c6a6d036f285fbaff5be9c601307ce9c5e71ca46#diff-a0da6d7c099dd9e2d22a140bbdf209f01f0f0394d19609d6d3e7c60a85ad760d)).

In summary it really appears that at this point it’s a test which is buying us nothing yet continuing to cost us — and given that, it seems like we’d be better off just removing it.

So, should I open a PR to remove the test?